bylaws.md

```diff
@@ -30,7 +30,7 @@ The Local shall meet in Convention annually during the first six months of the y
 
 ### Section 3: Special Meetings.
 
-A Special Meeting of the Local shall be called by the Local Council or by ten percent of members, with at least five days notice given to all members. The call to the Special Meeting shall specify the matters to be discussed therein and no other matter may be brought to the floor.
+A Special Meeting of the Local shall be called by the Local Council or by ten percent of members, with at least five days notice given to all members. The call to the Special Meeting shall specify the matters to be discussed therein and no other matter may be brought to the floor. The Local Council may also designate a scheduled informational meeting as a General meeting with at least 5 days notice given to all members.
```

Comment:
I don't get why the section on **special** meetings has a note talking about designating **informational** meetings as **general** meetings. It seems to me that this addition is either in the wrong section, or "informational" or "general" was supposed to say "special".

Reply:
Talking with Don, the right solution here is to change "General meeting" to say "Special Meeting".